dash_html.py

```diff
@@ -0,0 +1,384 @@
+# Copyright 2024 D-Wave
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This file stores the HTML layout for the app."""
+from __future__ import annotations
+
+from dash import dcc, html
+
+from app_configs import (
+    CHECKLIST,
+    DESCRIPTION,
+    DROPDOWN,
+    MAIN_HEADER,
+    RADIO,
+    SLIDER,
+    SOLVER_TIME,
+    THEME_COLOR_SECONDARY,
+    THUMBNAIL,
+)
+from src.enums import SamplerType
+
+SAMPLER_TYPES = {SamplerType.HYBRID: "Quantum Hybrid", SamplerType.CLASSICAL: "Classical"}
```

Comment:
I'm not sure if we want to always keep the "Hybrid" and "Classical" options. Might be that the demo instead uses the QPU directly, or doesn't compare to classical. Maybe change `src/enums.py` into a clearly customizable settings file by adding a description and comments with examples of how to use it?

Reply:
I updated this but let me know if this is better or what you think could make it better https://github.com/dwave-examples/template-dash/pull/2/commits/c455082d70a480fac66fc4c758629f971bf4bda1